.github/workflows/ci.yml

```diff
@@ -29,29 +41,28 @@ jobs:
             -Dparallel-test-playback \
             -Pdev \
             -f sdk/spring/pom.xml
+    - name: Build Spring Cloud Azure Experimental libraries with Maven
+      run: |
+          cd azure-sdk-for-java
           mvn clean install -Dmaven.javadoc.skip=true -DskipTests \
-            -Dcheckstyle.skip=true \
-            -ntp \
-            -Dspotbugs.skip=true \
-            -Drevapi.skip=true -Djacoco.skip=true \
-            -Dparallel-test-playback \
-            -Pdev \
-            -f sdk/spring-experimental/pom.xml
+                      -Dcheckstyle.skip=true \
+                      -ntp \
+                      -Dspotbugs.skip=true \
+                      -Drevapi.skip=true -Djacoco.skip=true \
+                      -Dparallel-test-playback \
+                      -Pdev \
+                      -f sdk/spring-experimental/pom.xml
+    - name: Build Spring Cloud Azure BOM with Maven
```

Comment:
bom already included in `sdk/spring/pom.xml`. 
https://github.com/Azure/azure-sdk-for-java/blob/feature/spring-boot-3/sdk/spring/pom.xml#L69